t/t2800-jobs-cmd.t

```diff
@@ -571,6 +571,20 @@ test_expect_success 'flux-jobs --format={runtime:0.3f} works' '
 	[ "$(grep -E "\.[0-9]{3}" runtime-dotRI.out | wc -l)" = "15" ]
 '
 
+test_expect_success 'flux-jobs --format={contextual_time} works' '
+	flux jobs --filter=pending -c1 -no "{contextual_time:h}" \
+		>ctx_timeP.out &&
+	flux jobs --filter=running -c1 -no "{contextual_time:h}" \
+		>ctx_timeR.out &&
+	flux jobs --filter=completed -c1 -no "{contextual_time:h}" \
+		>ctx_timeCD.out &&
+	echo "300.0" >duration.expected &&
+	test_cmp duration.expected ctx_timeP.out &&
+	test_must_fail test_cmp duration.expected ctx_timeR.out &&
+	test_must_fail test_cmp duration.expected ctx_timeCD.out &&
+	grep -E "\.[0-9]{3}" ctx_timeCD.out
```

Comment:
this grep is b/c the `{runtime}` tests above formatted with `0.3f`.  I suppose could have a spurious failure if some tests runtime was 1 second on the dot?  So perhaps update to "0.3f" too for consistency.  Or maybe just check runtime < 300.0?

Reply:
Ooh, good catch.

Reply:
Ok, fixed that one!